pkg/qemu-ga/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.5
```

Comment:
Where is that? If it is `tools/alpine` then isn't that a compile image (with compilers and a mirror in it). Since this is a runtime container I want it to be pretty minimal and not pull all that in. If `qemu-ga` was statically linked this would say `FROM scratch`, but it has library dependencies and working from an alpine base is the easiest way to resolve those.